packages/mint-components/src/components/tax-and-cash/sqm-indirect-tax-form/sqm-indirect-tax-form.feature

```diff
@@ -0,0 +1,81 @@
+@owner:andy @author:andy
+Feature: Indirect Tax Form
+
+  Background: A user has submitted their personal information in Tax Form Step One
+    Given a user is on the Indirect Tax Form
+
+  @minutia
+  Scenario Outline: Different indirect tax inputs are shown depending on the country of a participant
+    When <option> is selected based on based the participant <country> from step 1
+    Then different <inputs> appear
+
```

Comment:
@Locrian24 @AndiLuo I am not sure if the behaviour is right here. Similar to what Noad said and my other comment. Need to look into the Impact UI handling more.